tests/software/salmon/index/main.nf

```diff
@@ -5,8 +5,7 @@ nextflow.enable.dsl = 2
 include { SALMON_INDEX } from '../../../../software/salmon/index/main.nf' addParams( options: [:] )
 
 workflow test_salmon_index {
-    genome        = file("${launchDir}/tests/data/genomics/sarscov2/genome/genome.fasta", checkIfExists: true)
-    transcriptome = file("${launchDir}/tests/data/genomics/sarscov2/genome/transcriptome.fasta", checkIfExists: true)
-
-    SALMON_INDEX ( genome, transcriptome )
+    def genome_fasta     = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
```

Comment:
We don't really need to use `def` here. I removed it for all the other modules but there is also no harm in having it.

Reply:
I see, I'm getting the hang of it now. Thank you!